connexion/decorators/coroutines.py

```diff
@@ -0,0 +1,27 @@
+import asyncio
+import functools
+
+
+def get_request_life_cycle_wrapper(function, api, mimetype):
+    @functools.wraps(api.get_request)
+    def wrapper(*args, **kwargs):
+        request = yield from api.get_request(*args, **kwargs)
+        response = function(request)
+
+        if asyncio.iscoroutine(response):
+            response = yield from response
+
+        response = yield from api.get_response(response, mimetype,
```

Comment:
Does this return `ConnexionResponse` instance? Let's rename it to `connexion_resp` to distinguish native responses from connexion-wrapped ones